### ISSUE TITLE
Asking existing users to pay the same total amount $4.99 as new users pay.

### ISSUE THREAD

**Issue by ryouchinsa** [ryouchinsa/Rectlabel-support#64]
As a startup, we had to take a long time to obtain reputation and ask users to pay suitable price.
In Mar 2017, we released the first version of RectLabel for $4.99.
Nobody bought RectLabel except us.

We realized that RectLabel did not have enough reputation and features for paying $4.99.
We changed the price to be free and started updating to feature requests from our users.

RectLabel's price has been increased taking a long time.
$0    Mar 28 2017
$0.99 Dec 14 2017
$1.99 Jun  2 2018
$4.99 Aug  5 2018

Along this timeline, our users have been increasing and RectLabel has obtained reputation.
We think RectLabel is worth paying $4.99.

From the version 2.18, we started asking existing users(downloaded before Aug 5 2018) to pay the same total amount $4.99 as new users pay.
We are considering the amount you paid when you downloaded.
So that the amount you have to pay varies according to the download date.
$4.99 Mar 28 2017
$3.99 Dec 14 2017
$2.99 Jun  2 2018

We could survive on the image annotation tool page on wikipedia.
https://en.wikipedia.org/wiki/List_of_manual_image_annotation_tools

Because some users cited RectLabel on their papers and they were thought as independent, reliable sources.
We appreciate our users' contribution to increase our reputation.

We are a startup. 
We have to build a great product and monetize to survive among the competitors.
Otherwise we cannot continue running and updating RectLabel.

We hope you could understand.

If you could have some opinions, please let us know here.

Best regards,
Ryo

**Comment by nathanaherne** [comment]
I think I paid $7.99 AUD only a month or so ago. I was fine with that. Your software does what it says it will do and it does it well. Your support is fast, you are receptive to opinions and you are adding new features. I've paid a lot more for software and gotten a lot less.

**Comment by cooleel** [comment]
Can not make the purchase after my 2-weeks trial. Have tried a couple of times, also a couple of times following as stated in the troubleshooting part. 
Any ideas?

**Comment by ryouchinsa** [comment]
Thanks for reporting the issue.

Could you show us the screenshot?
https://github.com/ryouchinsa/Rectlabel-support/issues/53

**Comment by Digital-Rabbit** [comment]
Can you sell an app on the App Store and then require the user to pay again? That doesn't seem fair. Is that really a policy supported by the App Store?

**Comment by ryouchinsa** [comment]
Every month we have feature requests from our users.
Last month, from a medical system company, we had a feature request for "Superpixel and brush tool".
The new feature has been almost done and this feature would be useful for many users of RectLabel.

![superpixel_brush](https://user-images.githubusercontent.com/1954306/50035718-1ebea780-0047-11e9-94b9-a90a7c7b2b3b.gif)

To continue updating RectLabel for feature requests from our users, we have to monetize from the users who use RectLabel most.
So that we decided to ask existing users to pay extra fee as an in-app purchase.

To be fair, we are considering the amount you paid when you downloaded.
So that the amount you have to pay varies according to the download date.
If the amount is different on your environment, please restore purchase from app menu.
$4.99 Mar 28 2017
$3.99 Dec 14 2017
$2.99 Jun 2 2018

Let us know your opinion.

This is the LICENSED APPLICATION END USER LICENSE AGREEMENT.
https://www.apple.com/legal/internet-services/itunes/dev/stdeula/

**Comment by ryouchinsa** [comment]
Thanks for your opinion.
Currently we are not planning the price up again.

We have to monetize from users who use RectLabel the most.
To be fair, we asked our existing users to pay the same total amount as new users pay.
All users of RectLabel are being asked to pay the same amount.

> It would make way more sense to add InApp purchases for the new features. 

These are the current feature list of RectLabel.
https://rectlabel.com/help

> I am not against paying money for applications or something like that, just the way you try to get new money is a bit strange and does not feel very trustworthy to me.

If you could decide the price list for these features, could you show us the price list example comfortable for you?
You might not need all features so that just decide the price list for features you need.
As a result, how much are you going to pay to RectLabel?

If we need, we would write another issue as "Discuss the price list for the feature list of RectLabel".
If this question does work for our users, we would refer to the price list and consider to change our business model.

**Comment by pietz** [comment]
Ryo, I think you're mixing different discussions that are independent of one another:

Is the app good? 
I think so. 

Is the app worth $5? 
I think so. 

Is the app worth even more? 
I think so.

Even if every single person on this planet agreed with this, it would still not be ok to charge existing customers the difference to the current price. Don't take my word for it as I'm not a lawyer but I'm pretty darn sure it's also **illegal**. Not to mention that it probably goes against the Apple App Store policies.

A product offering is something where you offer a product in exchange for a price that you set. If a potential customer agrees to these terms he/she can enter into a sales contract with you. This is a binding contract. They give you the money you ask for. They get the product you advertised.

While it's okay to charge more money for additional features, you **CANNOT** charge more money for existing features. You're changing the contract to your own benefit and cripple the software if the users don't pay. It doesn't matter if you sold the software far to cheap. That are the terms that you set.

I would have bought this application for >$20 but I'm not going to. Either you will get banned from the App Store or you will do the whole thing again in the future when you decide your app is worth more.

**Comment by ryouchinsa** [comment]
Thanks for your opinion.

The new update version 2.58 was released.
https://rectlabel.com/help#version_before_in_app_purchase

If you bought RectLabel before starting in-app purchase, you can ask us to send the version 1.96 which is just before the in-app purchase.
At first, restore purchase from app menu to make the app purchase receipt in your app to the latest one.
Then, zip and send RectLabel app(/Applications/RectLabel.app) to support@rectlabel.com.
We would check the original app version which you bought, the transaction id for your payment, and the date when your app purchase receipt was updated.
If we could confirm, we would send the version 1.96 to you.

![スクリーンショット 2019-03-13 6 56 06](https://user-images.githubusercontent.com/1954306/54239874-cfade300-455f-11e9-9984-e377a00a5f7c.png)

Before launching the version 1.96, be sure to "Export settings file for the old version" from app menu.
Otherwise when you launch the version 1.96, the current settings file would be cleared because the old version cannot read the current format.
Import the exported settings file after you launch the version 1.96.

![スクリーンショット 2019-03-13 6 54 38](https://user-images.githubusercontent.com/1954306/54239884-d8061e00-455f-11e9-992e-02e33344c11c.png)

Let us know your opinion.

**Comment by ryouchinsa** [comment]
If your team has 10 members, how many licenses did you buy at this moment?

For the sake of clarity, each device used serially or collectively by multiple users requires a separate license.
https://www.apple.com/legal/internet-services/itunes/us/terms.html …

Please give us your feedback.

**Comment by ryouchinsa** [comment]
In the latest version, existing users have a save feature.

For Pixels feature and Export feature, you can buy as in-app purchases.

We could improve RectLabel based on your opinions.
Thank you.

![purchase_extra](https://user-images.githubusercontent.com/1954306/59202258-0df68500-8bd7-11e9-97c4-ae1767062a86.jpg)
